r_basics_transform_data/r_basics_transform_data.md

```diff
@@ -514,7 +526,10 @@ filter(covid_testing,
 
 We added whitespace (carriage returns and indentation) above to make this a bit more readable, but you don't have to.
 
-<div class = "warning">
+
+<div class = "help">
+<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
```

Comment:
This entire section feels WAY too long to be a div box ... I think I meant to comment on this last time, but must have forgotten (or maybe meant to comment here instead of the line 450ish fix which, while I do think makes more sense as not a trouble shoot box, doesn't quite fit the magnitude of the comment I put there). 


"This is definitely WAAAAAY too long to be a troubleshoot box. This isn't a quick snippet of info about troubleshooting, this is moreso like substantial educational content about how something works. Perhaps it makes sense to put this in another issue/pulled out into another PR, but I don't think this really qualifies as a highlight box-able situation at all."

Reply:
agreed! how about as a subsection?